### PR TITLE
added a logout icon

### PR DESCRIPTION
**Pull request by WilsontheWolf**
cause y not?
image
image

### CODE REVIEW

File: .gitconfig [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[push]
```
Comment (WilsontheWolf) [inline .gitconfig:1]: this was for my editor dont want it git ignore it so I don't have to deal with it

**Comment by Alexejhero** [comment]
Bruh this indentation's messed up XD

**Comment by WilsontheWolf** [comment]
idfc

**Comment by Alexejhero** [comment]
The website layout will drastically change, so I can't merge this.

**Pull request closed without merging**